birdhouse/env.local.example

```diff
@@ -226,6 +251,9 @@ export POSTGRES_MAGPIE_PASSWORD=postgres-qwerty
 #                                   }
 #"
 
+# Activates mounting a tutorial-notebooks subfolder that has the same name as the spawned image on JupyterHub
+# export MOUNT_IMAGE_SPECIFIC_NOTEBOOKS=true
```

Comment:
Can you add an explanation that `MOUNT_IMAGE_SPECIFIC_NOTEBOOKS=true`, is only useful if there are more than one image in `DOCKER_NOTEBOOK_IMAGES` and `ENABLE_JUPYTERHUB_MULTI_NOTEBOOKS` is set with proper `c.DockerSpawner.image_whitelist` matching the images in `DOCKER_NOTEBOOK_IMAGES`.